gitlab/mixins.py

```diff
@@ -324,6 +325,8 @@ def _get_update_method(
         """
         if self._update_uses_post:
             http_method = self.gitlab.http_post
+        elif self._update_uses_patch:
+            http_method = self.gitlab.http_patch
```

Comment:
I will refactor this into a proper mapping so we just return `self._update_method`, defaulting to PUT. But for now, this was a smaller change